djangocms_versioning/admin.py

```diff
@@ -104,6 +118,7 @@ def get_readonly_fields(self, request, obj=None):
         return super().get_readonly_fields(request, obj)
 
     def has_change_permission(self, request, obj=None):
+        # TODO: This is possibly a bug
```

Comment:
TODO – if it is a bug please raise a ticket for it. Probably shouldn't include this comment in the release.